build/get_new_operator_sdk.sh

```diff
@@ -1,16 +1,30 @@
 #!/bin/bash
 
+set -x
+
 REL=$(dirname "$0")
+readlink -f ${REL}
+REL=$( readlink -f $(dirname "$0"))
+
 ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
 OS=$(uname | awk '{print tolower($0)}')
 VERSION="${1:-v1.5.0}"
```

Comment:
This can probably be done separately, one the rest is working.

In the stf-run-ci role, this is never hit AFAIK, since we always call generate bundle with a version arg

Reply:
This was probably part of the loki / logging work Jaromir originally did. For STF 2.0.0 I plan to strip all of this out, but that'll be a separate branch in the future.